### PR TITLE
Show an error when the template will inexplicably fail.

### LINKED ISSUE #930: The ASP.NET Core template, using .NET Framework, fails on VS 2015.

### PR DESCRIPTION
**Pull request by ILMTitan**
Disable the template dialog and show an error when no 1.0.0-preview version of .Net Core SDK is found.

Fixes #930 
pr-943-example

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 1]
```diff
@@ -2994,4 +2994,13 @@
     <value>gcloud components install {0}</value>
     <comment>Command to install a missing component. {0} is the name of the component and should be in lower case.</comment>
   </data>
+  <data name="NoDotNetCorePreviewFoundError" xml:space="preserve">
+    <value>Could not find a 1.0.0-preview version of .Net Core SDK.
```
Comment (amanda-tarafa) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: It should be .NET Core SDK (that's the official name)

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 2]
```diff
@@ -2994,4 +2994,13 @@
     <value>gcloud components install {0}</value>
     <comment>Command to install a missing component. {0} is the name of the component and should be in lower case.</comment>
   </data>
+  <data name="NoDotNetCorePreviewFoundError" xml:space="preserve">
+    <value>Could not find a 1.0.0-preview version of .Net Core SDK.
+Either intall the Visual Studio 2015 DotNetCore tooling, or upgrade to Visual Studio 2017.</value>
```
Comment (amanda-tarafa) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Same here I think, .NET Core

File: GoogleCloudExtension/ProjectTemplates/ProjectTemplate.Tests/ProjectTemplatesTests.cs [hunk 3]
```diff
@@ -102,74 +94,81 @@ public void AfterEach()
         }
 
         [TestMethod]
-        [DataSource(
-            "Microsoft.VisualStudio.TestTools.DataSource.CSV",
-            @"|DataDirectory|\" + TemplatesCsvFileName + ".csv",
-            TemplatesCsvFileName + "#csv",
-            DataAccessMethod.Sequential)]
-        public void TestCompile()
+        [DataRow("4", "Mvc")]
```
Comment (amanda-tarafa) [inline GoogleCloudExtension/ProjectTemplates/ProjectTemplate.Tests/ProjectTemplatesTests.cs]: Oh, I didn't want to use the CSV but this I will be definitely using!

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 1 above)
Comment (ILMTitan) [reply GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Done.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 2 above)
Comment (ILMTitan) [reply GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Done.

File: GoogleCloudExtension/ProjectTemplates/ProjectTemplate.Tests/ProjectTemplatesTests.cs (hunk 3 above)
Comment (ILMTitan) [reply GoogleCloudExtension/ProjectTemplates/ProjectTemplate.Tests/ProjectTemplatesTests.cs]: I just discovered that this works. Not sure if it is due to the update to MSTest V2 or not.

**Pull request merged**